python/py-soco/Portfile

```diff
@@ -41,10 +41,7 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-requests-mock
```

Comment:
the `pytest` depedency is already added as `pytest` is the default `python.test_framework`

Reply:
thanks, fixed